src/Keyboard.js

```diff
@@ -23,7 +22,7 @@ export default class Keyboard extends PureComponent {
 		leftButtons: [],
 		rightButtons: [],
 		isFirstLetterUppercase: false,
-		defaultLanguage: 'cyrrilic',
+		languages: [LatinLayout, CyrillicLayout],
```

Comment:
The default layout was cyrillic, can you place `CyrillicLayout` on the `0` pisition?